openff/bespokefit/schema/tasks.py

```diff
@@ -77,6 +77,10 @@ class Torsion1DTaskSpec(QCGenerationTask):
         10,
         description="The number of initial conformers to seed the torsion drive with.",
     )
+    sp_specification: Optional[QCGenerationTask] = Field(
+        None,
+        description="An extra optional specification used to compute the reference energy surface on the optimised geometries.",
+    )
```

Comment:
Thanks for the thoughtful comments on this and I totally agree with the use of a new simple `QCModel`, I have wanted this in qcsubmit for a while now and it would be very useful as would still get the program, method and basis validation, if we create the separate model here we would lose that without repeating the code though.  

I agree that reworking the naming does make more sense maybe having a `calculation_specification` and a `optimization_specification` would be best? I also like the proposed structure of the `optimization_model`.

 For hessian and geometry optimisations this would have a different meaning in that the geometry is pre-optimised with the `optimization_specification` and then fully  optimised with the `calculation_specification` and then an optional hessian calculation is performed. A hessian calculation not at a stationary point on that particular PES would not help our fitting otherwise.